sentry-graphql-22/src/main/java/io/sentry/graphql22/SentryInstrumentation.java

```diff
@@ -0,0 +1,163 @@
+package io.sentry.graphql22;
+
+import graphql.ExecutionResult;
+import graphql.execution.instrumentation.InstrumentationContext;
+import graphql.execution.instrumentation.InstrumentationState;
+import graphql.execution.instrumentation.parameters.InstrumentationCreateStateParameters;
+import graphql.execution.instrumentation.parameters.InstrumentationExecuteOperationParameters;
+import graphql.execution.instrumentation.parameters.InstrumentationExecutionParameters;
+import graphql.execution.instrumentation.parameters.InstrumentationFieldFetchParameters;
+import graphql.schema.DataFetcher;
+import io.sentry.SentryIntegrationPackageStorage;
+import io.sentry.graphql.ExceptionReporter;
+import io.sentry.graphql.SentryGraphqlInstrumentation;
+import io.sentry.graphql.SentrySubscriptionHandler;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.TestOnly;
+
+@SuppressWarnings("deprecation")
+public final class SentryInstrumentation
+    extends graphql.execution.instrumentation.SimpleInstrumentation {
+
+  /**
+   * @deprecated please use {@link SentryGraphqlInstrumentation#SENTRY_SCOPES_CONTEXT_KEY}
+   */
+  @Deprecated
+  public static final @NotNull String SENTRY_SCOPES_CONTEXT_KEY =
```

Comment:
I think the two deprecated constants here could be removed, as the v22  is a new module, wdyt?

Reply:
I was thinking, customers would simply replace the dependency and package name and could be back to a working state without having to look for the constants. I'd keep it and remove it in the next major.